Microsoft.Toolkit.Uwp.UI.Controls.DataGrid/CollectionViews/CollectionView.cs

```diff
@@ -1258,7 +1258,7 @@ private enum CollectionViewFlags
         internal const string IsCurrentBeforeFirstPropertyName = "IsCurrentBeforeFirst";
         internal const string IsCurrentAfterLastPropertyName = "IsCurrentAfterLast";
 
-        // since there's nothing in the uncancelable event args that is mutable,
+        // since there's nothing in the uncancel-able event args that is mutable,
```

Comment:
Mmmh not sure this is the correct version 🤔
Whether we should pick the 1 or 2 Ls version is debatable (US for 1 L, UK for 2Ls), but "cancel-able" doesn't seem right. For consistency for other usages of the term in the toolkit, we should just stick with the US spelling here, so I'd say the previous "uncancelable" version was actually right?
```suggestion
        // since there's nothing in the uncancelable event args that is mutable,
```